integration_tests/src/main/python/spark_init_internal.py

```diff
@@ -166,6 +167,14 @@ def _handle_event_log_dir(sb, wid):
         .config('spark.eventLog.enabled', True) \
         .config('spark.eventLog.compression.codec', event_log_codec)
 
+def _handle_ivy_cache_dir(sb, wid):
+    if os.environ.get('SPARK_IVY_CACHE_ENABLED', str(True)).lower() in [
```

Comment:
what is the purpose of this flag?

Reply:
it tries to match above. so by default auto-gen the tmp ivy cache folder if xdist run, 
but we could also allow people to decide if they want to set the ivy-related config like use their own ivySettings

I was refer to https://github.com/NVIDIA/spark-rapids/blob/branch-22.10/integration_tests/src/main/python/spark_init_internal.py#L139-L143